ccmlib/node.py

```diff
@@ -772,7 +772,11 @@ def matches(expected, actual):
 
         total = 0
         for line in tail:
-            ks, cf, num = tail_pattern.search(line).groups()
+            print(f"'{line}'")
```

Comment:
using logging...

Reply:
also maybe we want a unit test for this parse function

Reply:
> using logging...

sorry, left over from debugging, already removed